### ISSUE TITLE
New dark mode is too dark, same as high contrast

### ISSUE THREAD

**Issue by LiamDawe** [mastodon/mastodon#29634]
### Steps to reproduce the problem

1. Look on Mastodon Social
2. Try the Dark or High Constrast themes
3. They're now basically the same

### Expected behaviour

They should be different

### Actual behaviour

They're the same

### Detailed description

The new themes have rolled out on Mastodon Social, and frankly the Dark theme is a big step backwards and introduces eye-strain. I'm not sure if it's supposed to look like it does, but it seems the High Contrast them is exactly the same as Dark now?

The original Dark theme was just fine overall on colour and contrast. This is a big step backwards.

### Mastodon instance

mastodon.social

### Mastodon version

v4.3.0-nightly.2024-03-18

### Browser name and version

Vivaldi

### Operating system

Linux

**Comment by Me4Linux** [comment]
I agree, the new Dark Theme is much more arduous to read. What was wrong with the previous colour scheme?

**Comment by leahneukirchen** [comment]
This is really uncomfortable to look at, please also offer a toned down version.
